### ISSUE TITLE
Create test suite

### ISSUE THREAD

**Issue by noppanit** [goldenapples/Co-Authors-Plus-Roles#18]
It would be nice if we have some test coverage.

**Comment by goldenapples** [comment]
You mean for the Javascript? Yup. Definitely needs it.

**Comment by noppanit** [comment]
Yes Javascript. You can choose whatever framework you're familiar with.

**Comment by danielbachhuber** [comment]
> You can choose whatever framework you're familiar with.

We should use the framework we already use, as we'll be maintaining this long-term.

**Comment by noppanit** [comment]
ok. Right now, we just Jasmine and PhantomJs for headless testing and Grunt. For mocking framework we pretty much use Jasmine Spy. But feel free to use Chai or anything that you're familiar with.
